### ISSUE TITLE
Create release v0.2.0

### ISSUE THREAD

**Issue by RandomDefaultUser** [mala-project/mala#204]
As mentioned in the Slack: It would be good to have a release for all the changes that happened since v0.1.0 and use it as a basis for the first DOI of this repository. The following things should/will be included in the release:

- [ ] Fixed installation instructions and OAT part of installation #176,#169 , #185
- [ ] Improved examples and made LDOS based examples runnable #191, #192 and #193
- [ ] Fixed direct string concat for file interaction and replaced it with `path` functions #195 and #197 
- [ ] Improved optuna hyperparameter optimization (ensemble objectives, band energy as validation loss, distributed optimization, performance study); there is still some improvements to be made here, but the current version works well
- [ ] Fixed several things regarding documentation and citation #200, #188, #108
- [ ] Added check ensuring that QE-MD generated input files adhere to PBC #183 
- [ ]  Implemented visualization via tensorbard #163 
- [ ] Stylistic improvements (import ordering fixed #179, TODOs converted to issues or resolved #174, replaced unncesseary `get_data_repo()` function
- [ ] Added bump version #203

**Comment by DanielKotik** [comment]
Super nice list!

**Comment by RandomDefaultUser** [comment]
All done, let's go to a PR :)
